packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -268,15 +262,16 @@ function ETHBalance({ on, prefix = '' }: { on: NetworkType; prefix?: string }) {
 
 function TokenBalance({
   forToken,
+  balance,
   on,
   prefix = ''
 }: {
   forToken: ERC20BridgeToken | null
+  balance: BigNumber | null
```

Comment:
```suggestion
  balance: BigNumber
```
i think it can stay as it is but note that selectedTokenBalances.l1 & .l2 are always of type `BigNumber`